src/server/lib/middleware/rateLimit.ts

```diff
@@ -66,9 +66,10 @@ export const rateLimiterMiddleware = async (
 
   if (ratelimitBucketTypeIfHit) {
     // Cut down the size of the access token so we don't make the log line too long.
-    const truncatedAccessToken = rateLimitData.accessToken
-      ? rateLimitData.accessToken.substring(0, 6)
-      : '';
+    const truncatedAccessToken =
+      typeof rateLimitData.accessToken === 'string'
+        ? rateLimitData.accessToken?.substring(0, 6)
```

Comment:
There won't be any need for the optional chaining syntax here as the type is known.

Reply:
👍